README.md

```diff
@@ -1,54 +1,82 @@
 # IBM Cloud SDK for Power Cloud
-To be updated
-power-go-client provides the Go implementation for operating the IBM Power Cloud platform to build LPARS .
-Installing
 
-Install the SDK using the following command
+The power-go-client project provides the Go SDK for IBM® Power Systems™ Virtual Server.
 
-```shell
-go get github.com/IBM-Cloud/power-go-client
-```
+## Prerequisites
+
+- An [IBM Cloud Account](https://cloud.ibm.com/registration)
+- An IBM Cloud [IAM API key](https://cloud.ibm.com/docs/account?topic=account-userapikey) or [Token](https://cloud.ibm.com/docs/account?topic=account-iamtoken_from_apikey) allow the SDK to access your account.
+- Go version 1.17 or above.
+
+## Install
+
+Install the SDK using the following methods.
 
-Update the SDK to the latest version using the following command
+### `go get` command
```

Comment:
I am fairly new to Go but I thought go get was being deprecated in Go 1.17 . . .
`Starting in Go 1.17, installing executables with go get is deprecated. go install may be used instead. In Go 1.18, go get will no longer build packages; it will only be used to add, update, or remove dependencies in go. mod . Specifically, go get will always act as if the -d flag were enabled.`

Reply:
`go get` is used for fetching the code in plain language.
`go install` is used to build and install packages in the user module.

`go get` is not getting deprecated but "installing executables with go get is deprecated".